alii.ash

```diff
@@ -1,268 +1,223 @@
-import <zlib.ash>
 void main(string settings) {
-
-string[int] argument;
-argument = split_string(settings, " ");
-
-string[string] available_choices = {
-  "coffee":"",
-  "ascend":"",
-  "gyou":"",
-  "cs":"",
-  "lunch":"",
-  "smoke":"",
-  "post":"",
-  "gloop":"",
-  "cloop":"",
-  "To make another option, copy another line of this and change just this text field":"",
-};
-
-foreach key in argument{  
-  foreach it in available_choices{
-    if(contains_text(to_lower_case(argument[key].to_string()), it)){
-      print(`Running choice {it}!`, "teal");
-      available_choices[it] = "true";
-    }
-  }
-}
-
-/* The above is just a fancy way of running multiple options at once haha. So you can run `alii coffee ascend gyou lunch smoke` all at once and take a nap after */
-//Disabling all but leaving intact.
-/*
-if (available_choices["all"].to_boolean()){
-  print("Running All. Whole day loop has been activated!");
-  foreach it in available_choices{
-    available_choices[it] = "true";
-  }
-}
-*/
-if (available_choices["post"].to_boolean()){
-  print("Running post run leg. Lunch and Smoke enabled!");
-  foreach it in $strings[lunch, smoke]{
-    available_choices[it] = "true";
-  }
-}
-
-if (available_choices["gloop"].to_boolean()){
-  print("Running post run leg. Lunch and Smoke enabled!");
-  foreach it in $strings[coffee, ascend, gyou, lunch, smoke]{
-    available_choices[it] = "true";
-  }
-}
-
-if (available_choices["cloop"].to_boolean()){
-  print("Running post run leg. Lunch and Smoke enabled!");
-  foreach it in $strings[coffee, ascend, cs, smoke]{
-    available_choices[it] = "true";
-  }
-}
-
-if (available_choices["coffee"].to_boolean()) {
-  Coffee();
-}
-
-if (available_choices["ascend"].to_boolean()){
-  Ascend();
-}
-
-if (available_choices["gyou"].to_boolean()) {
-  Gyou();
-}
-
-if (available_choices["cs"].to_boolean()) {
-  Cs();
-}
-
-if (available_choices["lunch"].to_boolean()) {
-  Lunch();
-}
-
-if (available_choices["smoke"].to_boolean()) {
-  Smoke();
+	// task keywords
+	boolean[string] available_choices;
+	foreach task in $strings[
+		coffee,
+		ascend,
+		gyou,
+		cs,
+		lunch,
+		smoke
+	] available_choices[task] = false;
+	// abbreviations
+	string[string] abbreviations = {
+		# "all": "coffee ascend gyou cs lunch smoke",
+		"post": "lunch smoke",
+		"gloop":"coffee ascend gyou lunch smoke",
+		"cloop":"coffee ascend cs lunch smoke"

```

Comment:
The cs loop does not run lunch. Lunch is only used for post gyou to levelup to 13 using NEP and setup a few other things for garbo.